docs/integrations/public-ip.md

```diff
@@ -0,0 +1,25 @@
+# Pipes' Public IP Addresses
```

Comment:
If we're already in the Pipes docs, do we need `Pipes'` here?

Reply:
Yeah it wasn't necessary, removed `Pipes'`